server/serverConstants.js

```diff
@@ -99,6 +99,10 @@ E.DEFAULT_INDEX_PAGE = { envVar: 'DEFAULT_INDEX_PAGE', jsonPath: 'defaultPageInd
 
 E.SHOW_TOS = { envVar: 'SHOW_TOS', jsonPath: 'showTos', defaultValue: false, parser: parseBool };
 
+E.SUBSCRIBER_RECEIVED_FRAMERATE = { envVar: 'SUBSCRIBER_RECEIVED_FRAMERATE', jsonPath: 'OpenTok.subscriberReceivedFramerate', defaultValue: 15 };
```

Comment:
Please remove the subscriber-related stuff from this PR. We will look into adjusting the preferred frame rate dynamically in the client.